### PR TITLE
Move "running on other node" info to debug/cluster log

### PR DESCRIPTION
**Pull request by HeinrichHartmann**

### CODE REVIEW

File: src/noit_check_tools.h [hunk 1]
```diff
@@ -151,7 +151,7 @@ MTEV_HOOK_PROTO(check_postflight,
 
 #define BAIL_ON_RUNNING_CHECK(check) do { \
   if(!noit_should_run_check(check, NULL)) { \
```
Comment (postwait) [inline src/noit_check_tools.h]: static mtev_log_stream_t debug_cluster= NULL; \
if(!debug_cluster) debug_cluster = mtev_log_stream_find("debug/cluster"); \
mtevL(debug_cluster, "Check %s is running on another node\n", check->name); \

**Comment by HeinrichHartmann** [comment]
Updated

File: src/noit_check_tools.h [hunk 2]
```diff
@@ -149,9 +149,12 @@ MTEV_HOOK_PROTO(check_postflight,
                 void *, closure,
                 (void *closure, noit_module_t *self, noit_check_t *check, noit_check_t *cause))
 
+static mtev_log_stream_t noit_cluster_debug = NULL;
```
Comment (postwait) [inline src/noit_check_tools.h]: Put this in the if... it's not good practice to declare a static file variable in a header.  You want static function scope.

**Comment by HeinrichHartmann** [comment]
yes, yes, yes. Just noted I pushed an old version. Fixed now. Sry.

**Comment by HeinrichHartmann** [comment]
Sorry. I have one more question here, if I initialize the variable to NULL we will always do the lookup again. If I don't, I am not sure if it NULL the first time we see it.

**Comment by postwait** [comment]
That's not how static initialization works.  That initialization only every happens once.

**Comment by HeinrichHartmann** [comment]
Cool. Then I'll merge this as is. Thx for the explanation.

**Pull request merged**